packages/react-native/ReactCommon/react/renderer/textlayoutmanager/TextMeasureCache.h

```diff
@@ -141,7 +141,7 @@ inline bool areAttributedStringFragmentsEquivalentLayoutWise(
         rhs.parentShadowView.layoutMetrics));
 }
 
-inline size_t textAttributesHashLayoutWise(
```

Comment:
This was the human mistake; it's a hash for `AttributedString::Fragment`, not for `TextAttributes`